src/reducers/auth.js

```diff
@@ -2,7 +2,7 @@ import * as types from '../actions';
 
 const initialState = {
   loading: false,
-  authorized: window.localStorage.authorized || false
+  authorized: window.localStorage.authorized || null
```

Comment:
The thing is, it's `null` when the first page is loaded, so the message doesn't show. After a failed login attempt, it turns into `false`, and the error message is shown.